aclk/agent_cloud_link.c

```diff
@@ -1280,24 +1280,33 @@ void *aclk_main(void *ptr)
     struct netdata_static_thread *static_thread = (struct netdata_static_thread *)ptr;
     struct netdata_static_thread *query_thread;
 
+#if defined( DISABLE_CLOUD ) || !defined( ENABLE_ACLK)
```

Comment:
I'm confused by this. Does the check only work if you disable the cloud?

Reply:
It's a bit redundant, I've removed the check. If either flag is set then a rebuild is required to activate the ACLK so we kill the thread.